app/build.gradle

```diff
@@ -7,6 +7,7 @@ apply from: '../versioning.gradle'
 
 ext {
     VERSION_NAME = "4.2.0"
+    USE_ORCHESTRATOR = project.hasProperty('orchestrator') ? project.property('orchestrator') : false
```

Comment:
Nice! Does this change the speed of execution? Can't tell from travis, it's still running.

Reply:
I doubt it will make it any faster. 😄 

But will report back after a few runs with some indication on impact.

Reply:
Speed seems roughly the same. Comparing with the last few successful runs from `develop`, they are all between 19.5 mins and 23.5 mins. Similar range for the last few runs from this branch.